pkg/runtimeproxy/server/types/docker_types.go

```diff
@@ -19,17 +19,28 @@ package types
 import (
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/api/types/network"
+
+	"github.com/koordinator-sh/koordinator/cmd/koord-runtime-proxy/options"
 )
 
 const (
 	ContainerTypeLabelKey       = "io.kubernetes.docker.type"
 	ContainerTypeLabelSandbox   = "podsandbox"
 	ContainerTypeLabelContainer = "container"
 	SandboxIDLabelKey           = "io.kubernetes.sandbox.id"
+
+	RuntimeProxySkipLabelKey = "runtimeproxy.koordinator.sh/is-hookserver"
```

Comment:
I think this mechanism should not be strongly bound to the hook server. All pods that need skip can follow this special protocol.

so node.koordinator.sh/skip-hookserver may be better.